tests/integrations/sqlalchemy/test_sqlalchemy.py

```diff
@@ -191,14 +191,6 @@ def processor(event, hint):
     # Some spans are discarded.
     assert len(event["spans"]) == 1000
 
-    # Some spans have their descriptions truncated. Because the test always
```

Comment:
this adds some useless non-determinism from the serializer, so I'm just killing it.